doc/Changelog.md

```diff
@@ -3,6 +3,15 @@ Changelog {#changelog}
 
 # Release 1.2 (git master)
 
+* [90](https://github.com/BlueBrain/Tide/pull/90):
+  The fullscreen mode has some new features:
+  - Users can now enlarge and pan contents (without the need to use the zoom)
+  - A double-tap toggles between *adjusted* and *maximized* fullscreen
```

Comment:
again confused. maximized fullscreen sounds wrong: why is fullscreen not maximized by default? If it's meant to replace the focus mode, did you remove it then?